src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoidinlineconditionals/InputAvoidInlineConditionals.java

```diff
@@ -4,9 +4,7 @@
 
 */
 
-package com . puppycrawl
-    .tools.
-    checkstyle.checks.coding.avoidinlineconditionals;
```

Comment:
we are not testing these lines. Do not see any "7" usage, so it should be aligned.